package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "MittHelsingborg",
```

Name of the app here sound be lowercase. I always get a warning here.
[RNUH](https://react-native-community.github.io/upgrade-helper/?from=0.64.1&to=0.66.2) agrees too.
```suggestion
  "name": "mitthelsingborg",
```

---

Not sure if this is used in the App for display when it starts up?

---

I agree that it should be changed to lowercase `mitthelsingborg`.

I wonder if it maybe should match the app bundle ID, however we are using different IDs for Android (`com.mitthelsingborg`) and iOS (`works.hbg.MittHelsingborg`)...so maybe not.

@dannil76 the package.json name is not used for display in the app, however it is used as part of the User Agent string we send on API requests:

```javascript
import { name, version } from "../../package.json";
...
const userAgent = `${name}/${version}/${Platform.OS}/${Platform.Version}`;
```